keps/sig-apps/2185-random-pod-select-on-replicaset-downscale/README.md

```diff
@@ -376,6 +376,7 @@ n/a
 
 - 2021-01-06: Initial KEP submitted
 - 2021-05-07: Updated KEP for graduation to beta
+- 2024-05-21：Updated KEP for graduation to GA
 
 ## Drawbacks
```

Comment:
Since this KEP is very old, we likely are missing PRR questions. Can you please check?

Reply:
According to the template, two missing questions were added, and the format was modified. Is this okay?